patches/components-os_crypt-os_crypt_mac.mm.patch

```diff
@@ -1,15 +0,0 @@
-diff --git a/components/os_crypt/os_crypt_mac.mm b/components/os_crypt/os_crypt_mac.mm
-index 49c22a9a5000851474421dc5eb9eb37422cda952..469c480164728d0577bc68aa48335b717c30c9e3 100644
---- a/components/os_crypt/os_crypt_mac.mm
-+++ b/components/os_crypt/os_crypt_mac.mm
-@@ -81,6 +81,10 @@ crypto::SymmetricKey* GetEncryptionKey() {
-     crypto::MockAppleKeychain keychain;
-     password = keychain.GetEncryptionPassword();
-   } else {
-+    base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
```

Comment:
This was likely only used when importing cookies... which we don't do anymore. We could test an import from Chrome that has saved passwords if we wanted to verify that still works (usually user gets prompted to put in keychain, so that passwords can be read/imported)